docs/.gitignore

```diff
@@ -1,2 +1,5 @@
-apache-airflow-providers-*
-apache-airflow
+_*
```

Comment:
Underscore? Is it a silly question asking what is this ignored for?

Reply:
It was for `_api`, `_build`, and `_doctree` generated by sphinx, but I since decided that better to generate all those in `generated` folder.

Reply:
Will remove

Reply:
In fact this .gitignore is not needed any more at all. The docs folder does not have any content generated any more - everything is either `_api` in local "doc" directories or the result is generated in `generated` folder and .gitignored there. I will also look if we can generate `_api`  elsewhere. I do not like `_api` files generated in the sources.